packages/flutter/lib/src/widgets/image.dart

```diff
@@ -606,13 +889,16 @@ class Image extends StatefulWidget {
 class _ImageState extends State<Image> {
   ImageStream _imageStream;
   ImageInfo _imageInfo;
+  ImageChunkEvent _loadingProgress;
   bool _isListeningToStream = false;
   bool _invertColors;
+  int _frameNumber;
+  bool _wasSynchronouslyLoaded;
 
   @override
   void didChangeDependencies() {
     _invertColors = MediaQuery.of(context, nullOk: true)?.invertColors
-      ?? SemanticsBinding.instance.accessibilityFeatures.invertColors;
+        ?? SemanticsBinding.instance.accessibilityFeatures.invertColors;
```

Comment:
we shouldn't rely on direct access to the binding unless we also register for notification of changes

Reply:
How does one register for notification of changes?

Reply:
From  https://master-api.flutter.dev/flutter/semantics/SemanticsBinding/accessibilityFeatures.html:

> To listen to changes to accessibility features, create a WidgetsBindingObserver and listen to didChangeAccessibilityFeatures.

Reply:
Done